cmd/bundlectl/main.go

```diff
@@ -16,18 +16,23 @@
 package main
 
 import (
-	"context"
-	"flag"
-	"os"
-
-	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands"
+  "context"
+  "flag"
+  "os"
+  "github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands"
 )
 
+var BUNDLECTL_VERSION string       // This value gets statically set in go compilation.  See the bazel rules. 
```

Comment:
One thing that's not clear to me: If you `go build` or `go install` this binary, does it no longer work? That would be unfortunate in my opinion. For basically all Kubernetes tools / binaries, you can `go install` them.

Reply:
It should. 
I'm not sure how to deal with the absolute url references to GoogleCloudPlatform to test this on my branch/repo.

It's a build flag you have to specify.   Not sure if there's a way to specify default flags.   
Once we get this on the main branch I can check it out.  If there's not an obvious way, we'll just hardcode the string.